paths.go

```diff
@@ -0,0 +1,128 @@
+package quamina
+
+import (
+	"fmt"
+	"strings"
+)
+
+const PathSeparator = "\n"
+
+type pathsIndex struct {
+	root bool
+
+	nodes map[string]*pathsIndex
+
+	// fields map from it's name to it's full path (as specific in quamina.Field)
```

Comment:
s/it's/its/ because "it's" == "it is". Like we say "he's" for "he is" but not "hi's".

Reply:
Thanks, changed!